### ISSUE TITLE
Handlers don't support lmi_port variable

### ISSUE THREAD

**Issue by Eskotus** [IBM-Security/isam-ansible-roles#101]
I noticed that when ever I use something other than 443 for lmi_port most of the handlers fail. Looks like they use a variable named "port" and passes it to lmi_port and if that is missing it omits the variable altogether causing the isam module to revert to default 443.

I changed port to lmi_port and it seems to pick up the lmi_port in vars files. I could put in a pull request about this unless there is some reason to use a different variable in the handlers.

**Comment by ram-ibm** [comment]
It used to be lmi_port - got changed recently. Yes that needs to be fixed.

**Comment by Eskotus** [comment]
So what would be the best workaround for transition phase before the new variable "port" is used in all roles? Include both variables in var files?

**Comment by ram-ibm** [comment]
Yes -please use both variables for now. I plan to change the port to lmi_port in the handlers. Have also reached out to the folks who made the change.
